panther_ioc_rules/log4j_exploit_iocs.py

```diff
@@ -0,0 +1,21 @@
+from panther_iocs import LOG4J_EXPLOIT_IOCS
+
+
+def nested_value_search(data, search_value):
+    if not hasattr(data, "__iter__"):
+        return False
+    if isinstance(data, list):
+        return any(nested_value_search(item, search_value) for item in data)
+
+    if isinstance(data, dict):
+        return any(nested_value_search(v, search_value) for v in data.values())
+
+    return search_value in data
+
+
```

Comment:
Optional, but is there an appropriate `title` or `dedup` function to use here? Maybe something that references `p_log_type` so you get a different alert per log type?

Reply:
p_log_type is a good idea, I was struggling to come up with what the right thing was